### PR TITLE
update release notes for airgap

### PR DESCRIPTION
**Pull request by songawee**
connect https://github.com/waffleio/waffle.io-takeout/issues/161

### CODE REVIEW

File: releases/2.x.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+_Released 2016.06.14_
+
+_Supported GH:E versions: GH:E 2.2+_
+
+#### Manual Migration Step Required
+- For this release we require you to run a manual, one-line script to migrate Replicated, our on-premises platform, from 1.x to 2.x: http://docs.replicated.com/docs/installing-replicated#migrating-from-replicated-v1
+
+#### Features
+
+- Enable users to install Waffle Takeout to via an Airgap installation
```
Comment (burtonjc) [inline releases/2.x.md]: extraneous "to"

File: INSTALL.md [hunk 2]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
 
 ![Step 8](doc/screenshots/6.png)
 
+### Airgap Installation
+
+For installations without external internet access or are behind a proxy, you can install Waffle Takeout using the following steps.
```
Comment (burtonjc) [inline INSTALL.md]: "or are" sounds a little awkward. Maybe "For installations who don't have easy access to the internet"

File: INSTALL.md [hunk 3]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
...
+DOCKER_OPTS="--storage-opt dm.basesize=20G"
+```
+
+Once that change is made, restart the docker daemon:
+
+```bash
+sudo service docker stop
+sudo rm -rf /var/lib/docker
+sudo service docker start
+```
```
Comment (burtonjc) [inline INSTALL.md]: We should confirm this works across all supported distros.

File: INSTALL.md [hunk 4]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
...
+
+#### 2. The Airgap package is what is downloaded from the Airgap link we provide you. You can `shift-click` in your browser to download the file or you can use something like `wget`.
```
Comment (burtonjc) [inline INSTALL.md]: Where do we provide that link? Might be helpful to tell people where.

File: INSTALL.md [hunk 5]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
...
+```bash
+wget --trust-server-names -O <your_file_name>.airgap "<link_we_provide>"
+```
+
+#### 3. Once you have the host machine up and running and have installed docker and replicated, upload the `.airgap` package onto the host machine and remember the path to the file. One way is to use `scp`.
+
+
+```bash
+scp -i ~/.ssh/your_key.pem /path/to/your_airgap_file.airgap ubuntu@<host_machine_url>:/path/to/upload/to
+```
```
Comment (burtonjc) [inline INSTALL.md]: This is run not on the host machine

File: INSTALL.md [hunk 6]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
...
+
+> Note: You may need to run the following commands to restart the Management Screen UI (Ubuntu/Debian)
+
+```
+sudo service replicated restart
+sudo service replicated-ui restart
+sudo service replicated-operator restart
+```
```
Comment (burtonjc) [inline INSTALL.md]: this is run on the host machine. Might want to be explicit about where to run these commands.

File: INSTALL.md [hunk 7]
```diff
@@ -75,6 +75,51 @@ $ curl -sSL https://takeout.waffle.io/get | sudo bash
...
+
+#### 4. Continue following the [Replicated Airgap install docs](http://docs.replicated.com/docs/airgapped-installations) to locate the package on the host machine and upload your license in the management ui.
+
+#### 5. Follow the above steps for installing Waffle Takeout starting from Step 6
```
Comment (burtonjc) [inline INSTALL.md]: let's make "Step 6" a link to the actual step.

File: INSTALL.md [hunk 8]
```diff
@@ -92,6 +137,20 @@ $ sudo apt-get install replicated replicated-ui replicated-agent replicated-upda
 
 You can check for new app releases by going to the management interface dashboard `https://waffle.company.com:8800` and clicking on the 'Check Now' button. If an update is available you will be able to read the release notes and install the update.
 
+#### Updating an Airgap installation
+
+To update an Airgap installation, you will need to download a new package via a link that we provide you and upload it to the server location specified in the "Console Settings" in the management ui.
```
Comment (burtonjc) [inline INSTALL.md]: Let's clarify where we provide that link.

File: INSTALL.md [hunk 9]
```diff
@@ -92,6 +137,20 @@ $ sudo apt-get install replicated replicated-ui replicated-agent replicated-upda
...
+
+![Airgap Settings](doc/screenshots/7.png)
+
+##### On Host Machine
+```bash
+cd /home/ubuntu/updates
+wget --trust-server-names -O <your_update>.airgap "<link_we_provide>"
+```
```
Comment (burtonjc) [inline INSTALL.md]: Given that this is an airgap install, they probably cannot do a `wget` from the host machine. They will probably have to do the `wget` on another machine outside the network, the `scp` it onto the host machine.

File: INSTALL.md (hunk 3 above)
Comment (songawee) [inline INSTALL.md]: added note that this is Ubuntu/Debian specific

File: INSTALL.md (hunk 4 above)
Comment (songawee) [inline INSTALL.md]: Made a bold section in the intro to address this

File: INSTALL.md (hunk 5 above)
Comment (songawee) [inline INSTALL.md]: specified that it's run on your computer

File: INSTALL.md (hunk 6 above)
Comment (songawee) [inline INSTALL.md]: specified that this is a host machine command

File: INSTALL.md (hunk 8 above)
Comment (songawee) [inline INSTALL.md]: clarified in opening section of installation

**Comment by burtonjc** [comment]
👍

**Pull request merged**